src/api/dataclasses/cf_response_objects.py

```diff
@@ -50,8 +80,8 @@ class CFDataResponse:
 
```

Comment:
Can probably get rid of this if we're just going to pass the error up to be handled.